easybuild/easyconfigs/l/libidn/libidn-1.36-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'libidn'
+version = '1.36'
```

Comment:
See #12663 - it looks like we should use a 2.x version?

Reply:
Hm yes. I wonder if we should introduce a new `libidn2` for it instead? https://www.gnu.org/software/libidn/#libidn2

Reply:
Is that needed? Other than `wget` I cannot see `libidn` used as a dependency in recent easyconfigs?

Reply:
Well.. it just seems neater; they are updated independently https://ftp.gnu.org/gnu/libidn/ (in fact, 1.36 is newer than 2.3.0). The library is called libidn2, the binary is called idn2. I see debian and centos are packaging libidn2 separately (and both might be required, my centos machine has both at least)

Reply:
I really think something like this #12665 is most appropriate

Reply:
OK. That makes sense.